index.bs

```diff
@@ -2628,26 +2628,26 @@ instance on which it was called.
 The <dfn method for=IDBDatabase>transaction(|storeNames|,
 |mode|)</dfn> method, when invoked, must run these steps:
 
-1. If |mode| parameter is not {{"readonly"}} or {{"readwrite"}},
-    [=throw=] a [=TypeError=].
-
-2. If this method is called on {{IDBDatabase}} object for which an
+1. If this method is called on {{IDBDatabase}} object for which an
```

Comment:
on an {{IDBDatabase}}?

FWIW, I liked instance a bit better than object. Also, I think you can get away without either.